locales/en/data-classes.ftl

```diff
@@ -4,6 +4,7 @@ account-balances = Account balances
 address-book-contacts = Address book contacts
 age-groups = Age groups
 ages = Ages
+appointments = Appointments
```

Comment:
Please add a quick note that this refers to doctor appointments. I have a feeling a few language might confuse this with something else.